contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -185,181 +197,203 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
     }
 
     /**
-      * @notice Set the maximum number of validators to stake for the node operator #`_id` to `_stakingLimit`
-      */
-    function setNodeOperatorStakingLimit(uint256 _id, uint64 _stakingLimit) external
+     * @notice Set the maximum number of validators to stake for the node operator #`_id` to `_stakingLimit`
+     */
+    function setNodeOperatorStakingLimit(uint256 _id, uint64 _stakingLimit)
+        external
         authP(SET_NODE_OPERATOR_LIMIT_ROLE, arr(_id, uint256(_stakingLimit)))
         operatorExists(_id)
     {
         require(operators[_id].stakingLimit != _stakingLimit, "NODE_OPERATOR_STAKING_LIMIT_IS_THE_SAME");
         _increaseKeysOpIndex();
-        operators[_id].stakingLimit = _stakingLimit;
-        emit NodeOperatorStakingLimitSet(_id, _stakingLimit);
+        _setNodeOperatorStakingLimit(_id, _stakingLimit);
+        _updateTotalAvailableKeysCount();
     }
 
     /**
-      * @notice Report `_stoppedIncrement` more stopped validators of the node operator #`_id`
-      */
-    function reportStoppedValidators(uint256 _id, uint64 _stoppedIncrement) external
+     * @notice Report `_stoppedIncrement` more stopped validators of the node operator #`_id`
+     */
+    function reportStoppedValidators(uint256 _id, uint64 _stoppedIncrement)
+        external
         authP(REPORT_STOPPED_VALIDATORS_ROLE, arr(_id, uint256(_stoppedIncrement)))
         operatorExists(_id)
     {
         require(0 != _stoppedIncrement, "EMPTY_VALUE");
         operators[_id].stoppedValidators = operators[_id].stoppedValidators.add(_stoppedIncrement);
         require(operators[_id].stoppedValidators <= operators[_id].usedSigningKeys, "STOPPED_MORE_THAN_LAUNCHED");
```

Comment:
could save some gas by avoiding storage reads of `stoppedValidators`

Reply:
fixed